projects/rawkode-academy/web/src/components/partner/vs-conference.astro

```diff
@@ -11,18 +11,18 @@
 		</h2>
 		<p class="mb-3 font-light text-gray-500 dark:text-gray-400 sm:text-xl">
 			Of course you have, we all have. We spend tens of thousands of dollars
-			paying the conference organizer, flourishing a booth, flying the team
+			paying the conference organizer, setting up a booth, flying the team
```

Comment:
I wasn't sure what the original intent was here, so I took my best guess..."furnishing" maybe?

Reply:
I was using "flourishing" as in to flourish:

> a bold or [extravagant](https://www.google.com/search?sca_esv=847412b30e358a21&sca_upv=1&sxsrf=ADLYWIJb5WDEu0wlCbQFCB0byTTaX6Mfww:1723151506550&q=extravagant&si=ACC90nwKPQWKXvO0LWGU61hOTgoD_alYrCjb3L0T-l27n8_1QoeoK77QJmKMLzzjPlpET8jhxaQbqTyu9ah-gr3HSifHkcIu6IYCaP8Ufu6o15A20Lr5GVo%3D&expnd=1&sa=X&ved=2ahUKEwiFn8TLp-aHAxWSxAIHHdhAKkUQyecJegUINRDgAQ) [gesture](https://www.google.com/search?sca_esv=847412b30e358a21&sca_upv=1&sxsrf=ADLYWIJb5WDEu0wlCbQFCB0byTTaX6Mfww:1723151506550&q=gesture&si=ACC90nypsxZVz3WGK63NbnSPlfCBgaT53pweyfGqen8tS0IXOxc3rlwwZ2w1GCbtx4lopDK38sdcbQ3Pxk3v7ZtjDR0T_oCT5g%3D%3D&expnd=1&sa=X&ved=2ahUKEwiFn8TLp-aHAxWSxAIHHdhAKkUQyecJegUINRDhAQ) or action, made especially to attract attention.

Reply:
right, that's usually used as an intransitive verb, and the transitive use would usually mean holding in your hand (a la, "flourishing a baton"), so it might confuse people a bit, but you're also welcome to use forms of words however the hell you want :laughing:.

Reply:
I think you're right. Keep this change